test/server.cpp

```diff
@@ -211,6 +214,53 @@ ResourceCollection all200Resources()
   return concat(resources200Compressible, resources200Uncompressible);
 }
 
+TEST(indexTemplateStringTest, emptyIndexTemplate) {
+  const int PORT = 8001;
+  const ZimFileServer::FilePathCollection ZIMFILES {
+    "./test/zimfile.zim",
+    "./test/corner_cases.zim"
+  };
+
+  std::unique_ptr<ZimFileServer>   zfs_;
+  zfs_.reset(new ZimFileServer(PORT, ZIMFILES, ""));
+  EXPECT_EQ(200, zfs_->GET("/")->status);
+  zfs_.reset();
```

Comment:
This can be better written like :
```cpp
auto zfs_ = sd::unique_ptr<ZimFileServer>(new ZimFileServer(PORT, ZIMFILES, ""));
EXPECT_EQ(200, zfs_->GET("/")->status);
```
No need to call reset at the end as the unique_ptr will delete the allocated ZimFileServer (this is the only purpose of unique_ptr, it would be sad to not us it correctly)

(Same for the test `indexTemplateCheck`)